src/lambda.group.ts

```diff
@@ -0,0 +1,45 @@
+import { Context } from 'aws-lambda';
+import express from 'express';
+import { LambdaHandler } from 'index';
+import { DefaultPathParamsPattern, DefaultPort, LocalLambda } from './local.lambda';
+
+
+export interface LambdaConfig {
+  handler: LambdaHandler;
+  context?: Context;
+  enableCORS?: boolean;
+  // default binary content-type headers or override
+  binaryContentTypesOverride?: string[];
+  pathParamsPattern ?: string;
+  requestContext?: Record<string, any>;
+}
+
+
+export interface LocalLambdaGroupConfig {
+  lambdas: LambdaConfig[];
+  port?: number;
+  defaultPath?: string;
+}
+
+export class LocalLambdaGroup {
+  lambdas: LambdaConfig[] = [];
+  app: express.Application;
+  port: number;
+  defaultPath: string;
+
+  constructor(config: LocalLambdaGroupConfig) {
+    this.lambdas = config.lambdas;
+    this.app = express();
+    this.port = config.port ?? DefaultPort;
+    this.defaultPath = config.defaultPath ?? DefaultPathParamsPattern;
+  }
+
+  run(): void {
+    this.lambdas.forEach(lambda => {
+      const localLambda = new LocalLambda(lambda, this.app, this.defaultPath);
+      localLambda.createServer();
+      this.app = localLambda.app;
```

Comment:
one more question here, any reason why we need to re-assgin the `localLambda.app` back to the `this.app` on each round ? I thought `this.app` is already initialized at the contructor?

Reply:
Nice catch, I will address to this as this is not required.

Reply:
Nice catch, I will address to this as this is not required.